### PR TITLE
Arquillian docs improvements

### PR DESCRIPTION
**Pull request by OndroMih**
Port of https://github.com/payara/Payara-Community-Documentation/pull/142

### CODE REVIEW

File: docs/modules/ROOT/pages/documentation/ecosystem/arquillian-containers/payara-server-managed.adoc [hunk 1]
```diff
@@ -87,34 +121,24 @@ proxy configuration).
...
 
 
 NOTE: The context root that will be used to run the tests is also retrieved automatically from the DAS server. If you do not have a `sun-web.xml`, `glassfish-web.xml` or `payara-web.xml` file in your web application, Payara will use the name of your deployment without the extension as context root. The same rule is applied for enterprise applications if there is no `application.xml` file. The jar test-deployments are treated as a web application. You can use the above standard GlassFish / Payara XML files as normal to declare your context root to be applied. You should consider the above description to avoid any conflict with your already deployed web or enterprise applications in your administrative domain. 
 
-The easiest way to provide these system properties to the extension runtime is by using an arquillian.xml file placed on the test classpath.
+[[examples]]
+== Examples
+
+[[configure-example]]
+=== Exampe Configuration
```
Comment (fturizo) [inline docs/modules/ROOT/pages/documentation/ecosystem/arquillian-containers/payara-server-managed.adoc:136]: Typo at "Exampe"

File: docs/modules/ROOT/pages/documentation/ecosystem/arquillian-containers/payara-server-managed.adoc [hunk 2]
```diff
@@ -125,7 +149,72 @@ The easiest way to provide these system properties to the extension runtime is b
...
+            <my.admin.password>mypassword</my.admin.password>
+        </systemPropertyVariables>
+    </configuration>
+</plugin>
+----
+
+[[automatic-download-example]]
+### Downloading Payara Server automatically from Maven
+
+You can automate downloading and installing Payara Server with maven so that the Arquillian plugin can be automatically configured and used without installing Payara Server manually.
```
Comment (fturizo) [inline docs/modules/ROOT/pages/documentation/ecosystem/arquillian-containers/payara-server-managed.adoc:179]: Bit of a hard read, better to state: "You can configure your Maven project to automatically download and install the Payara Server needed by the Arquillian plugin"

**Review by fturizo — APPROVED** [review]
Looks good, a couple of minor changes needed.

File: docs/modules/ROOT/pages/documentation/ecosystem/arquillian-containers/payara-server-managed.adoc (hunk 1 above)
Comment (OndroMih) [reply docs/modules/ROOT/pages/documentation/ecosystem/arquillian-containers/payara-server-managed.adoc:136]:
```suggestion
=== Example Configuration
```

File: docs/modules/ROOT/pages/documentation/ecosystem/arquillian-containers/payara-server-managed.adoc (hunk 2 above)
Comment (OndroMih) [reply docs/modules/ROOT/pages/documentation/ecosystem/arquillian-containers/payara-server-managed.adoc:179]:
```suggestion
You can configure your Maven project to automatically download and install the Payara Server needed by the Arquillian plugin
```

File: docs/modules/ROOT/pages/documentation/ecosystem/arquillian-containers/payara-server-managed.adoc [hunk 3]
```diff
@@ -125,7 +149,72 @@ The easiest way to provide these system properties to the extension runtime is b
...
+            <my.admin.password>mypassword</my.admin.password>
+        </systemPropertyVariables>
+    </configuration>
+</plugin>
+----
+
+[[automatic-download-example]]
+### Downloading Payara Server automatically from Maven
+
+You can configure your Maven project to automatically download and install the Payara Server needed by the Arquillian plugin
```
Comment (OndroMih) [inline docs/modules/ROOT/pages/documentation/ecosystem/arquillian-containers/payara-server-managed.adoc:179]:
```suggestion
You can configure your Maven project to automatically download and install the Payara Server needed by the Arquillian plugin.
```

**Pull request merged**